pkg/block/indexheader/binary_reader.go

```diff
@@ -279,14 +273,16 @@ func newBinaryWriter(id ulid.ULID, cacheFilename string, buf []byte) (w *binaryW
 
 		// We use file writer for buffers not larger than reused one.
```

Comment:
What does this comment mean? Perhaps we could remove it?

Reply:
Umm I have no idea. Yeah let's remove the comment

Reply:
Updated the comment.